Gibe.DittoProcessors/Processors/LinkPickerAttribute.cs

```diff
@@ -14,7 +21,14 @@ public override object ProcessValue()
 				return null;
 			}
 
-			return JsonConvert.DeserializeObject<LinkPickerModel>(Value.ToString());
+			var link = JsonConvert.DeserializeObject<LinkPickerModel>(Value.ToString());
+
+			if (link.Id != default(int))
+			{
+				link.Url = _umbracoWrapper.TypedContent(link.Id).Url;
```

Comment:
What happens if the page has been deleted? I think this will error somewhere it can't be easily caught